app/server-utils/stampy.ts

```diff
@@ -114,20 +143,27 @@ const getCodaRows = async (
 
 const md = new MarkdownIt({html: true}).use(MarkdownItFootnote)
 
-const extractText = (markdown: string) => markdown?.replace(/^```|```$/g, '')
+// Sometimes string fields are returned as lists. This can happen when there are duplicate entries in Coda
```

Comment:
this was fun. The offending coda row is now fixed, but I left this here just in case - otherwise it's enough for someone to create 2 gdocs with the same name to blow up everything